src/fetch_family_allowance.py

```diff
@@ -0,0 +1,91 @@
+import datetime
```

Comment:
Would you mind organizing a bit this import section? In PEP 8 we have something like:

> Imports should be grouped in the following order:
> * standard library imports
> * related third party imports
> * local application/library specific imports
> 
> You should put a blank line between each group of imports.

We don't have to be strict, but this could be inspirational source ; )